lib/src/utils.dart

```diff
@@ -90,17 +90,22 @@ String winstonToAr(BigInt winston) {
 
 /// Safely get the error from an Arweave HTTP response.
 String getResponseError(Response res) {
-  if (res.headers['Content-Type'] == 'application/json') {
-    Map<String, dynamic> errJson = json.decode(res.body);
-
-    if (errJson['data'] != null) {
-      return errJson['data'] is Map
-          ? errJson['data']['error']
-          : errJson['data'];
+  try {
+    if (res.headers['Content-Type'] == 'application/json') {
+      Map<String, dynamic> errJson = json.decode(res.body);
+
+      if (errJson['data'] != null) {
+        return errJson['data'] is Map
+            ? errJson['data']['error']
+            : errJson['data'];
+      }
     }
-  }
 
-  return res.body;
+    return res.body;
+  } catch (e) {
+    print('Unknown upload error. Retrying...');
+    return 'unknown error';
```

Comment:
Can we print and return the actual error message?

Reply:
Okay